docs/developer/getting-started/running-kibana-advanced.asciidoc

```diff
@@ -52,6 +52,11 @@ non-dev version and accepts any of the
 https://www.elastic.co/guide/en/kibana/current/settings.html[standard
 settings].
 
+[discrete]
+=== Using an Alternate YML File
+
+To run Kibana with an alternate yml file, use the `--config` option to specify the path to the desired yml file, along with the `--dev` option. For example: `bin/kibana --config=config/my_config.yml --dev`
```

Comment:
`bin/kibana` is the command you'd use to startup a Kibana distributable. Since this document is targeting developers, we should update this to reflect how we run Kibana from source locally.

Note that `yarn start` automatically includes the `--dev` flag, so I removed that portion from my suggestion below:

```suggestion
To run Kibana with an alternate yml file, use the `--config` option to specify the path to the desired yml file. For example: `yarn start --config=config/my_config.yml`
```